packages/create-wmr/src/index.js

```diff
@@ -99,7 +99,9 @@ sade('create-wmr [dir]', true)
 	.parse(process.argv);
 
 async function scaffold({ cwd, fields }) {
-	await templateDir(resolve(__dirname, '../tpl'), resolve(cwd || '.'), fields);
+	const to = resolve(cwd || '.');
+	await templateDir(resolve(__dirname, '../tpl'), to, fields);
+	await fs.rename(resolve(to, 'gitignore'), resolve(to, '.gitignore'));
```

Comment:
Shouldn't this rename from .npmignore?

Reply:
That could work, but I renamed `.gitignore` to `gitignore`. Could revert that, and just rename the `.npmignore`. 

Actually that's probably a better solution. That way, in the repo, it at least looks right.

Reply:
Updated. I do like that more.